packages/react-refresh/src/ReactFreshBabelPlugin.js

```diff
@@ -10,7 +10,7 @@
 export default function(babel, opts = {}) {
   if (typeof babel.getEnv === 'function') {
```

Comment:
Should we change this check too?

Reply:
@gaearon Good callout, thanks! It's fixed now.